zipline/utils/test_utils.py

```diff
@@ -581,5 +581,6 @@ def create_data_portal_from_trade_history(env, tempdir, sim_params,
 
 
 class FakeDataPortal(object):
-    def __setattr__(self, key, value):
-        pass
+
```

Comment:
I usually add one, really this change was because I deleted and retyped the class body and did it out of habit.

In Python's own documentation `class` seems to be done with or without a new line interchangibly.